std_msgs/Float64  msgA
geometry_msgs/Twist  msgB